content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,129 @@
+The *Hint Providers* then use this when making the resource allocation decisions.
+
+### Configuration
+
+The Topology Manager is an alpha feature in Kubernetes v1.16 and will be disbabled by default behind a feature gate. 
+
+The Topology Manager currently:
+* Works on a nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/)
+* Works on Pods in the `Guaranteed` QOS Class {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}. 
+If these conditions are met, Topology Manager will align CPU and device requests.
+
+Topology Manager has two allocation policies, that are set via a Kubelet flag `--topology-manager-policy`.
+There are two supported policies:
+
+*`none`: the default, which means no Topology Management will take place and Kubelet will preform
+as previously.
+*`best-effort`: which means the Topology Manager will provide the best fit allocation for the container
+but will not fail the pod in the event this is not possible.
+
+### None policy
+
+This is the default policy and does not preform any topology alignment.
+
+### Best-effort policy
+
+The `best-effort` policy makes calls for each incoming container to the 
+Hint Providers to provide their resource availablity.
+Using this information, the Topology Manager stores the 
+best NUMA Node affinity for that container. If the affinity is not preferred, 
+Topology Manager will store this and admit the pod to the node.
+
+The *Hint Providers* can then use this information when making the 
+resource allocation decision.
+
+### User Experience
+
+Consider the containers in the following pod specs:
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+```
+
+This pod runs in the `BestEffort` QoS class because no resource `requests` or
+`limits` are specified.
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        memory: "200Mi"
+      requests:
+        memory: "100Mi"
+```
+
+This pod runs in the `Burstable` QoS class because requests are less than limits.
+
+Neither of these pod specifications would be considered by the Topology Manager *best-effort* policy.
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        memory: "200Mi"
+        cpu: "2"
+        example.com/device: "1"
+      requests:
+        memory: "200Mi"
+        cpu: "2"
+        example.com/device: "1"
+```
+
+This pod runs in the `Guaranteed` QoS class because `requests` are equal to `limits`.
+
+This pod would be considered by the Topology Manager. In this case the CPU Manager Static policy would be consulted to send back available CPUs and their Topology, as well as this the Device Manager would be consulted to provide the Topology of example.com/device.

```

Comment:
I tried to rephrase this closer to project style. Here's my suggestion:
```suggestion
Topology Manager would consider this Pod. The Topology Manger consults the CPU Manager Static, which returns available CPUs and their topology. Topology Manager also consults Device Manager  to provide the Topology of `example.com/device`.
```

Reply:
Changed, let me know what you think